src/generateRandomNumber.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const numberToGuess = [];
+
+while (numberToGuess.length !== 4) {
+  const randomNumber = Math.floor(Math.random() * 10);
+
+  if (numberToGuess.indexOf(randomNumber) === -1) {
+    numberToGuess.push(randomNumber);
+  }
+}
+
+module.exports = { numberToGuess };
```

Comment:
Better to create function in this file that returns random number and export it
Then you can just call this function in main file to get random number